nixos/doc/manual/release-notes/rl-1703.xml

```diff
@@ -34,6 +34,9 @@ following incompatible changes:</para>
       <literal>gtk</literal>, <literal>gtkmm</literal> and several others.
       Now you need to use versioned attributes, like <literal>gnome3</literal>.
     </para>
+    <para>
+      All font packages have been moved into `nixpkgs.fonts`.
```

Comment:
Note that this is not Markdown, so please use something like `<literal>` instead of backticks.